db/migrate/20170330161442_change_institution_defaults.rb

```diff
@@ -0,0 +1,48 @@
+class ChangeInstitutionDefaults < ActiveRecord::Migration
+  def change
+    change_table :institutions do |t|
+      t.change :flight, :boolean, default: nil
+      t.change :correspondence, :boolean, default: nil
+      t.change :gibill, :integer, default: nil
+      t.change :yr, :boolean, default: nil
```

Comment:
The relevant search filter code could accept `true`, `false`, or `nil`. However, the functionality of the GIBCT should be only to search (using the checkboxes) for `true` values. There should be no mechanism to search for a boolean that is `nil` or `false`. In other words, filtering should only be `true` or not `true` - this is how the current tool works.